src/ephysatlas/utils.py

```diff
@@ -0,0 +1,34 @@
+import hashlib
+from pathlib import Path
+from typing import Any, Dict
+
+
+def setup_output_directory(params: Dict[str, Any]) -> Path:
+    """Set up the output directory structure and change to it."""
```

Comment:
Here it would be nice to add the result of the `tree` command on one of the actual datasets that we have to show the user what is the expected output from the function.

This tree output (for which we have a mockup in #11 ) should also be part of the documentation.